packages/fluentui/react-component-ref/test/Ref-test.tsx

```diff
@@ -1,39 +1,236 @@
-import { Ref, RefFindNode, RefForward } from '@fluentui/react-component-ref';
-import { shallow } from 'enzyme';
+import { Ref } from '@fluentui/react-component-ref';
```

Comment:
you're replacing almost all the tests in this file, is it possible to write them in testing-library ?

Reply:
It's just copy-paste, not really replacing...